modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/controller/PortletImportControllerImpl.java

```diff
@@ -130,8 +129,11 @@
  * @author Máté Thurzó
  */
 @Component(
-	property = "model.class.name=com.liferay.portal.kernel.model.Portlet",
-	service = {ExportImportController.class, PortletImportController.class}
+	property = {
+		"import.controller=true",
+		"model.class.name=com.liferay.portal.kernel.model.Portlet"
+	},
+	service = PortletImportController.class
```

Comment:
Hey @GabrielHenriquedev, the change in the tracker that supports this modification should also be in this commit. Thx